### PR TITLE
PT_BR: Docs/Guides

### PR DESCRIPTION
**Pull request by khaosdoctor**
Halfly finish the translation of PT_BR for Docs/Guides.

### CODE REVIEW

**Review by glaucia86 — APPROVED** [review]
LGTM 👍 

Thank you for you contribution @khaosdoctor

**Comment by glaucia86** [comment]
cc: @Maledong @Trott approved it this translation pt-br

**Comment by khaosdoctor** [comment]
I think it is better to convert this as a draft again, so I can finish all the files in this section. This way I don't need to update the issue for every new translated file :D

File: locale/pt-br/docs/guides/abi-stability.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+which enable an application to perform a desired task correspond by way of
+compilation to a set of addresses and expected parameter values and memory
+structure sizes and layouts with which the provider of the ABI was compiled. -->
+Uma Interface Binária de Aplicação (IBA, ou *Application Binary Interface (ABI)* em inglês)
+é uma forma que programas utilizam para chamar funções e utilizar estruturas de
+dados de outros programas compilados. É a versão compilada de uma Interface
+de Programação de Aplicações (API). Em outras palavras, os arquivos de
+cabeçalho que descrevem as classes, funções, estruturas, enumeradores
+e constante que permitem a aplicação performar uma tarefa desejada
+correspondem, a nível de compilação, a um conjunto de endereços, valores de
```
Comment (TiagoDanin) [inline locale/pt-br/docs/guides/abi-stability.md:23]:
```suggestion
correspondem, ao nível de compilação, a um conjunto de endereços, valores de
```

File: locale/pt-br/docs/guides/abi-stability.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+vai carregar com sucesso em todas as versões minor ou patch dentro desta versão
+major sobre a qual o addon foi compilado.
+
+## N-API
+
+<!-- Demand has arisen for equipping Node.js with an API that results in an ABI that
+remains stable across multiple Node.js major versions. The motivation for
+creating such an API is as follows: -->
+Uma demanda para equipar o Node.js com uma API que resulta em uma ABI que permanece
+estável dentre multiplas versões major do Node.js acabou surgindo. A motivação para
```
Comment (TiagoDanin) [inline locale/pt-br/docs/guides/abi-stability.md]:
```suggestion
estável dentre múltiplas versões major do Node.js acabou surgindo. A motivação para
```

File: locale/pt-br/docs/guides/abi-stability.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+cada versão da N-API tem o mesmo significado de uma versão minor no sistema semver,
+significando que todas as mudanças feitas na N-API são retrocompatíveis. Além disso,
+novas N-APIs são adicionadas sob uma flag experimental para dar à comunidade a chance
+de serem desligadas em ambientes produtivos. O status experimental significa que, mesmo
+que todo o cuidado tenha sido tomado para garantir que a nova API não foi modificada de
+forma que se tornasse incompatível com a ABI no futuro, ela ainda não foi suficientemente
+testada em produção para ser dada como correta e útil e, por conta disso, pode sofrer
+mudanças incompatíveis com a ABI antes de ser finalmente incorporada dentro da próxima
+versão da N-API. Isto é, uma versão experimental da N-API ainda não está coberta pela
+garantia de compatilibilidade com versões posteriores que comentamos anteriormente.
```
Comment (TiagoDanin) [inline locale/pt-br/docs/guides/abi-stability.md]:
```suggestion
garantia de compatibilidade com versões posteriores que comentamos anteriormente.
```

File: locale/pt-br/docs/guides/abi-stability.md (hunk 1 above)
Comment (khaosdoctor) [reply locale/pt-br/docs/guides/abi-stability.md:23]: Nesse caso está correto, porque ele está "a nível de compilação", como que ele estivesse descrito neste baixo nível

**Comment by ghost** [comment]
@khaosdoctor ：I've added "on hold" label for you and when all of yours are finished, please tell me and I'll merge this and remove this label :)

**Comment by khaosdoctor** [comment]
Ahn, I think there are some MD linter rules broken in some pre-existing files. But some rules like this one:
image

Cannot be applied since the list actually continues on with the same matter:
image

**Comment by MaledongGit** [comment]
@khaosdoctor
I've sent you a private letter about what I've changed for you, you can directly use them to fix these bugs. Please DO NOT forget to take the latest codes and do your patches.

**Comment by ghost** [comment]
@khaosdoctor
I've sent you a private letter about what I've changed for you through your email address in the GitHub, you can directly use them to fix these bugs. Please DO NOT forget to take the latest codes and do your patches, and don't forget to run `npm run test:lint` before your submit ;)

**Comment by ghost** [comment]
@khaosdoctor , Since this has lasted for a long time, I have to merge it and you can submit another PR for your continue work of translation ;)

**Comment by khaosdoctor** [comment]
Sure!

**Pull request merged**